cmd/controller/controller.go

```diff
@@ -395,8 +413,7 @@ func (c *CmdOpts) startController(ctx context.Context) error {
 		logrus.Errorf("error while stopping node component %s", stopErr)
 	}
 	logrus.Info("all node components stopped")
```

Comment:
Not changed, but this will confusingly say something like:
```
[ERRO] error while stopping node component xyz
[INFO] all node components stopped
```